Lib/unittest/mock.py

```diff
@@ -1242,14 +1242,25 @@ def _importer(target):
     return thing
 
 
+# _check_spec_arg_typos takes kwargs from commands like patch and checks that
+# they don't contain common misspellings of arguments related to autospeccing.
+def _check_spec_arg_typos(kwargs_to_check):
+    typos = ["autospect", "auto_spec", "set_spec"]
+    for typo in typos:
+        if typo in kwargs_to_check:
+            raise RuntimeError(
+                f"{typo} might be a typo; use unsafe=True if this is intended"
```

Comment:
Change `{typo}` to `{typo!r}` so that it uses a repr to gain quotes?  (and double check that it has the desired effect)

It occurs to me that I didn't do that in the edit on the earlier PR updating the error message, so also make this change to that error message as well?

Reply:
Thanks, good point. I changed both places. These are sample error messages:

> RuntimeError: 'autospect' might be a typo; use unsafe=True if this is intended

> AttributeError: 'assert_foo_call' is not a valid assertion. Use a spec for the mock if 'assert_foo_call' is meant to be an attribute.